saw-remote-api/src/SAWServer.hs

```diff
@@ -162,10 +170,29 @@ initialState =
                 , rwLaxArith = False
                 , rwWhat4HashConsing = False
                 }
-     return (SAWState emptyEnv bic [] ro rw)
+     return (SAWState emptyEnv bic [] ro rw M.empty)
 
 validateSAWState :: SAWState -> IO Bool
-validateSAWState _ = pure True
+validateSAWState sawState =
+  checkAll
+    [ CryptolServer.validateServerState cryptolState
+    , checkAll $ uncurry checkHash <$> M.assocs (view trackedFiles sawState)
```

Comment:
Given that we're working in `IO`, but `<$>` is happening on `[]`, it might be easier to read if `map` were written here. Or even a list comprehension. But this is a quite minor style point!

Reply:
Fixed by https://github.com/GaloisInc/argo/pull/70/commits/b61eb9ac2f1e790117f74931eaa8731cdc0d1352